.github/workflows/update-unused-messages.yml

```diff
@@ -0,0 +1,32 @@
+name: "[Check] remove unused messages and create PR"
+
+on:
+  workflow_dispatch:
```

Comment:
Is it necessary to add another action? You're also not using some naming convention for the file. We can add it to the same action but just add a trigger that checks if the action was scheduled to run to only do this instead of updating it.

Reply:
Yes, doing this every time you sync crowdin seems a bit better to me. 👍

Reply:
So what is the final idea? will it be run with the Crowding sync or with this workflow as well?

Reply:
Please add it to the Crowdin sync. It should run before the upload to Crowdin.

Reply:
The upload also deletes from Crowdin? I'm just thinking whether it would be dangerous if someone ran it without checking the unused strings prior

Reply:
Most likely not, except for the possibility of dynamic strings that are not marked as such.

Reply:
I added a checkbox as an input to confirm removing to make it safer